src/app/containers/ImageWithPlaceholder/index.jsx

```diff
@@ -96,6 +97,7 @@ const ImageWithPlaceholder = ({
             height={height}
             width={width}
             style={!isImgJpg ? { backgroundColor: C_GHOST } : null}
+            data-hero={isHero}
```

Comment:
Could be worthwhile doing something like `{...(isHero && { 'data-hero': true })}` here. I think having it as `data-hero:false` is fine, but I've ran into some issues with AMP not liking certain values set for attributes that are either "true" or nothing.